pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -38,9 +38,11 @@ def _strip_raw_data(tensor):
     meta_dict['type'] = "stripped"
     meta_dict['average'] = float(numpy.average(arr))
     meta_dict['variance'] = float(numpy.var(arr))
+    if not tensor.HasField("raw_data"):
+        tensor.raw_data = onnx.numpy_helper.from_array(arr, tensor.name).raw_data
```

Comment:
Now this code would be unnecessary since we don't use `tensor.raw_data` anymore:
```suggestion
```

Reply:
@impactaky Could you check this comment?
We are planning to cut v0.5.1 tomorrow, so if you would like to have this PR in the release, it would be great if you could reflect the suggestion before that. :bow:

Reply:
Sorry, for late reply :bow:
tensor.raw_data still needed, because onnx.external_data_helper.set_external_data require raw_data field
https://github.com/onnx/onnx/blob/1cc5e12f95c168146a912e3a6837d51f5d0f693b/onnx/external_data_helper.py#L83-L84

Reply:
Sorry I've forgot that!
Then this is resolved. Thank you for the explanation!